x-pack/test/functional/apps/advanced_settings/feature_controls/advanced_settings_security.ts

```diff
@@ -138,7 +138,8 @@ export default function({ getPageObjects, getService }: FtrProviderContext) {
       });
     });
 
-    describe('no advanced_settings privileges', () => {
+    describe('no advanced_settings privileges', function() {
+      this.tags(['skipCoverage']);
```

Comment:
Are there issues for the things we're skipping here?  Should we mention them in a comment?

Reply:
Yes, I was not able to run x-pack from master locally, but hopefully I can do it tomorrow and will report an issue.